cmd/configschema/go.mod

```diff
@@ -2,7 +2,7 @@
 // See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30187
 module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/configschema
 
-go 1.21.0
+go 1.21.9
```

Comment:
I raised this in https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/32356. I assume we still do not want this yet? cc @open-telemetry/collector-contrib-approvers

Reply:
Filed SAP/go-hdb/pull/136 and SAP/go-hdb/issues/135

Reply:
Fixed :rocket:

Reply:
I guess this PR still needs to wait for a latter version with https://github.com/SAP/go-hdb/pull/136